app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -470,7 +470,7 @@
   </div>
   <div class="field">
     <label>{{t 'Event Sub-topic'}}</label>
-    {{#ui-dropdown class='search selection' selected=data.event.subTopic onChange=(action (mut data.event.subTopic)) as |execute mapper|}}
+    {{#ui-dropdown class='search selection' allowAdditions=true selected=data.event.subTopic.name onChange=(action (mut data.event.subTopic)) as |execute mapper|}}
```

Comment:
We do not allow additionsFor this. The user must not be able to add their own subtopic.